app-misc/piper/piper-0.4.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
```

Comment:
Please also test it with Python 3.8.

Reply:
Same issue with python 3.8 as libratbag.

Reply:
Resolved.